plugins/python/_hexchat.py

```diff
@@ -64,16 +66,18 @@ def emit_print(event_name, *args, **kwargs):
         arg = args[i].encode() if len(args) > i else b''
         cstring = ffi.new('char[]', arg)
         cargs.append(cstring)
-    if time is 0:
+
+    if time == 0:
```

Comment:
Not sure the value of changing all the `is` usage, `==` is just slower.

Reply:
`is` is an identity check, so it only works between [-5 and 256](https://docs.python.org/3/c-api/long.html#c.PyLong_FromLong) as those are cached in the small integer cache. if a future change removes or changes the this could break. the `is` check pretty much does `id(a) == id(b)`, so its not really any slower afaik

Reply:
The world would explode if it was changed so it will never change.

Reply:
This is true. Not sure if it helps readability, though what it WILL do is cause inconsistencies if we ever do that check on something bigger than 256

Reply:
Alright